gcp/deploy.txt

```diff
@@ -0,0 +1,12 @@
+pulumi up
```

Comment:
Should this be used as a test script? or maybe should not have been committed?

Reply:
This file has already been deleted .. not sure how it surfaced here ..